### ISSUE TITLE
#2770: data_attributes.rst, refine `markdown` syntax in 'data_attribute.rst'

### ISSUE THREAD

**Issue by rahul-kumi** [jeff1evesque/machine-learning#2770]
Refine syntax of `data_attributes.rst` file which was overlooked in the issue:#2767

**Comment by rahul-kumi** [comment]
@jeff1evesque , could you clear the air, as to what is it you find should be refined?

**Comment by jeff1evesque** [comment]
I would ensure our current [`data_attributes`](https://github.com/jeff1evesque/machine-learning/blob/master/documentation/programmatic_interface/data_attributes.rst) conforms to the equivalent rst syntax guideline, which can be reviewed with the following sources, and examples:

- https://gist.github.com/dupuy/1855764
- https://raw.githubusercontent.com/scikit-learn/scikit-learn/master/doc/install.rst
  - corresponds to https://github.com/scikit-learn/scikit-learn/blob/master/doc/install.rst

In addition, I'd move `#### Data Attributes` to the top of the [`data_attributes`](https://github.com/jeff1evesque/machine-learning/blob/master/documentation/programmatic_interface/data_attributes.rst) file.

Since this issue is a `bug`, we should prefix the branch with`bug-`, instead of `feature-`.  Specifically, the following should be the commit style for this issue:

```bash
$ git checkout master
$ git pull upstream master
$ git pull origin master
$ git checkout -b bug-2770
## make your changes
$ git add /path/to/machine-learning/documentation/programmatic_interface/data_attributes.rst
$ git commit -m "#2770: data_attributes.rst, [your own comment]"
$ git push origin bug-2770
```

**Comment by jeff1evesque** [comment]
Closing this issue manually, since the corresponding pull request (https://github.com/jeff1evesque/machine-learning/pull/2775) forgot to include the keyword `resolves #2770`, in the issue body.
